src/tracker-component.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+import { getClientID } from '@paypal/sdk-client/src';
```

Comment:
can `getClientId` be undefined?

Reply:
the result of it? yes, but then the sdk won't work at all. `getClientId` itself though, no won't be undefined

Reply:
Nope.